wsl-vpnkit

```diff
@@ -120,6 +122,32 @@ check () {
     check_https "https://$CHECK_HOST"
 }
 
+start () {
+    # run cleanup to restore any leftover configuration changes
+    cleanup
+    wsl2tap_down
+    run
+    iptables_set A
+    check
+}
+
+monitor () {
+    while true
+    do
+        if ss -elx | grep -q $WSL_INTEROP; then
+            : #everything is going well, socket still valid, no action to do
+        else
+            echo "Something is going bad, socket $WSL_INTEROP broken, trying to repair..."
+            echo "killing all subprcesses from this wsl-vpnkit-script"
+            pstree -A -p $$ | grep -Eow "[0-9]+" | tail -n +2 | xargs kill
```

Comment:
Seems like there might be an issue with `pstree` command:
```
pstree -A -p $$

pstree: unrecognized option: A
BusyBox v1.35.0 (2022-11-19 10:13:10 UTC) multi-call binary.

Usage: pstree [-p] [PID|USER]

Display process tree, optionally start from USER or PID

        -p      Show pids
```

This is on the latest `wsl-vpnkit` imported distro `0.4.1`

Reply:
Removed the "A" option to work in an Alpine distro.
Works for me in my Ubuntu distro